src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -28,7 +28,8 @@ record TestInputs(
       String osVersion,
       OperatingSystemType os,
       SidecarInfo.VsCode vscode,
-      String vscodeVersionPrefix
+      String vscodeVersionPrefix,
+      String userAgent
   ) {
```

Comment:
I believe we'll also need to specify the OS architecture as part of the test inputs, and pass it in the properties as `OS_ARCH_KEY` (like how we do for `OS_NAME_KEY` and `OS_VERSION_KEY`). This way, we override the OS arch of the machine the test itself is running on.